srcalc/src/main/webapp/WEB-INF/views/fragments/booleanFragment.jsp

```diff
@@ -0,0 +1,9 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
```

Comment:
All these files should have a `.jspf` because they are fragments, not standalone JSP files.